src/main/kotlin/com/zepben/evolve/services/network/translator/NetworkProtoToCim.kt

```diff
@@ -450,6 +458,7 @@ fun toCim(pb: PBUsagePoint, networkService: NetworkService): UsagePoint =
         connectionCategory = pb.connectionCategory.takeIf { it.isNotBlank() }
         ratedPower = pb.ratedPower.takeIf { it != UNKNOWN_INT }
         approvedInverterCapacity = pb.approvedInverterCapacity.takeIf { it != UNKNOWN_INT }
+        pb.phaseCode?.name?.let { phaseCode = PhaseCode.valueOf(it) }
```

Comment:
todo: `pb.phaseCode` is not nullable. I am not sure why the IDE is not detecting this, but there is no need to wrap this in the `let`. In the future we will need to do proper processing of the `UNRECOGNIZED` values from all enums, which can fix this one up at the same time.